docs/linking.md

```diff
@@ -0,0 +1,12 @@
+# Developing with linked packages
+
+If you want to make changes to a package that Element Call depends on and see those changes applied in real time, you can create a link to a local copy of the package. Yarn has a command for this (`yarn link`), but it's not recommended to use it as it ends up modifying package.json with details specific to your development environment.
+
+Instead, you can use our little 'linker' plugin. Create a file named `.links.yaml` in the Element Call project directory, listing the names and paths of any dependencies you want to link. For example:
+
+```yaml
+matrix-js-sdk: ../path/to/matrix-js-sdk
+"@vector-im/compound-web": /home/alice/path/to/compound-web
```

Comment:
I am loving this.

Do I understand it correctly, that it just allows us to put this part of the package.json into its own file, so that we can git-ignore it?
pnpm has the same behaviour and I was wondering how this could be elegantly solved.
This is great!

Reply:
Can a plugin also create the a `.links.lock` file and add compose the actual lock file based on `yarn.lock`+`.links.lock`?

Reply:
This is resolved with the git hooks.